### PR TITLE
Add uri-stripper and destination-router filters

### PR DESCRIPTION
**Pull request by VinnyQ**
* uri-stripper, template and .pp config
* destination-router, template and .pp config

### CODE REVIEW

File: templates/destination-router.cfg.xml.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<destination-router xmlns='http://openrepose.org/repose/destination-router/v1.0'>
```
Comment (mwhahaha) [inline templates/destination-router.cfg.xml.erb]: This needs fixin'.  In 7 it's been converted to docs.openrepose.org but you'll need to keep 6 backwards compatibility.  You may want to look at how this is handled for other files.

File: templates/uri-stripper.cfg.xml.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<uri-stripper xmlns="http://docs.api.rackspacecloud.com/repose/uri-stripper/v1.0" rewrite-location="<%= @rewrite_location %>" token-index="<%= @token_index %>"/>
```
Comment (mwhahaha) [inline templates/uri-stripper.cfg.xml.erb]: Same here on the namespace.

**Comment by VinnyQ** [comment]
@mwhahaha  Thanks for catching that Alex.  Please review namespace fix when you have a chance.

**Comment by mwhahaha** [comment]
@VinnyQ looks ok, once the tests pass i'll merge

**Pull request merged**